common/components/forms/ImageCropUploadFormElement.jsx

```diff
@@ -45,9 +45,10 @@ class ImageCropUploadFormElement extends React.PureComponent<Props,State> {
   }
 
   render(): React$Node {
+    const aspect = 16 / 9;
```

Comment:
ImageCropUploadFormElement should also have a prop for aspect ratio, defaulting to 1:1 for now since that is the image format our existing cases (project and user thumbnail) use.